api-server/api-server-common/src/storage/impls/postgres/transactional/mod.rs

```diff
@@ -131,7 +131,7 @@ impl<'a> ApiServerPostgresTransactionalRo<'a> {
     pub async fn get_transaction(
         &mut self,
         transaction_id: Id<Transaction>,
-    ) -> Result<Option<(Option<Id<Block>>, SignedTransaction)>, ApiServerStorageError> {
+    ) -> Result<Option<(Option<Id<Block>>, TransactionInfo)>, ApiServerStorageError> {
```

Comment:
Please don't forget to bump the storage version when changing the internal format. It'll automatically wipe the database and start over.